packages/cli/src/util/get-latest-version/get-latest-worker.js

```diff
@@ -247,6 +247,14 @@ async function fetchDistTags(name) {
         });
         res.on('end', () => {
           try {
+            if (res.statusCode && res.statusCode >= 400) {
```

Comment:
Can you use (!res.ok) instead?

Reply:
This is using Node's built-in `https.get()` in which the handler receives a `ServerResponse` and that response object doesn't have an `ok` property.